### DISCUSSION TITLE
Azure CI/CD - Deploy, distribute and update

### DISCUSSION THREAD

**Discussion by MikeKatsoulakis** [Squirrel/Squirrel.Windows#1771] [Q&A]
I Cannot figure out how to use squirrel in my CI/CD process, how would I use the squirrel commands?
Also is there a way to make the update manager look for updates in Azure Blob Storage?

Thanks!

**Comment by vernou** [comment]
1. Download the package NuGet "Squirrel.Windows"
2. Unzip "Squirrel.Windows.npkg"
3. Get "<Unzipped>\tools\Squirrel.exe"

Currently, I did this manually on my CD server... but a script would be better.
I plan to do this script... one day.

If anyone has realized it beforehand, please share it.

**Reply by anaisbetts** [reply]
I usually use Cake to do this automatically
